### ISSUE TITLE
Links with brackets doesn't get auto tooltip

### ISSUE THREAD

**Issue by vaniwiki** [ComboStrap/combo#103]
I know you mentioned that you don't support brackets in links, so this is a long shot... But I have hundreds of thousands of links in brackets! And I would very much like them to work with auto tooltip, how they were working previously... Maybe that was with the separate AutoTooltip plugin, but none the less...

Example of my links: [[books:bg:4:34|[Bg. 4.34]]]. Any other way to make it work than changing all of them?

**Comment by gerardnico** [comment]
I can't change how dokuwiki parse the document unfortunately.


You can always disable it but you loose all the benefits.

**Comment by vaniwiki** [comment]
Wait, are you saying if I disable the link component, all the previous functionality will work as well? Like search highlights and so on...?

**Comment by gerardnico** [comment]
Because I take over / parse the link component to allow formatted content inside (and all the features listed in the page)

dokuwiki does not offer any nested component inside a link. It's an url and a text.

https://combostrap.com/content/the-link-component-x0oocdu2#square_bracket_not_allowed

**Comment by gerardnico** [comment]
> Wait, are you saying if I disable the link component, all the previous functionality will work as well? Like search highlights and so on...?

Nope. You will loose it all.

**Comment by vaniwiki** [comment]
Yes, I see now. But how then DokuWiki was doing that previously?

**Comment by vaniwiki** [comment]
I mean, I love working with ComboStrap! But I'm just scrambling to get my website working again before lack of funds force me to close. I already lost my previous server due to low bank balance, then I switched over to my development server, that was unfortunately not ready... The reason I bother to change, upgrade server, convert tens of thousands of documents into frontmatter compatible, redoing so much of the work I've already done, starting from scratch... It's because of ComboStrap and all its fantastic functionality (that is nowhere else to be found). The positive far outweight the small gripes I have with it, and sometimes there is also misunderstanding in writing. I hope I did not offend you earlier... I think your work is absolutely genius, and I hope to be able to work with it for a long time to come. Indeed, if I ever get funded, you will be the first one I'll show my appreciation! (In the form of a humble donation most likely, but still..)

Sorry for the rant. I'll stop here, before I get in trouble. Just wanted you to know I appreciate your work. Thanks again. :)

**Comment by vaniwiki** [comment]
Maybe it was the Autotooltip plugin that did it previously. If the plugin author could alter his plugin to work with ComboStrap descriptions, I could possibly go back to that one. Anyhow, that's another thing. Is there a way to escape it? I can alter links in my code, as long as they show up as brackets. Or html entity codes for the same? Wait, let me try...

**Comment by vaniwiki** [comment]
> Or html entity codes for the same? Wait, let me try...

Hah! How silly of me. Like you said...

**Comment by gerardnico** [comment]
Ouah if html entities work, you should post your answer here.

The things is the link allow nested syntax in the description. May be that was a bad decision and I should have created another component but I wanted also to be move compatible.

In the previous setup, you can do anything with JavaScript ;)

**Comment by vaniwiki** [comment]
> Ouah if html entities work, you should post your answer here.

No, they don't work. Like you said, DokuWiki just takes a url and a text. So there is no solution to this as of yet.

> In the previous setup, you can do anything with JavaScript ;)

I wish I knew programming... I only know css, wiki syntax and ComboStrap web markup. However, if I ever get my project funded, I will definitely hire someone to help with all the things I wish I knew. Then I could peacefully work on my wiki and archive content. And when that is up to standard, there are other archives that needs publishing.
